gremlin-test/features/map/AddVertex.feature

```diff
@@ -336,7 +336,7 @@ Feature: Step - addV()
       | marko |
     And the graph should return 2 for count of "g.V().has(\"name\",\"marko\")"
 
-  Scenario: g_V_asXaX_hasXname_markoX_outXcreatedX_asXbX_addVXselectXaX_labelX_propertyXtest_selectXbX_labelX_valueMapXtrueX
+  Scenario: g_V_asXaX_hasXname_markoX_outXcreatedX_asXbX_addVXselectXaX_labelX_propertyXtest_selectXbX_labelX_valueMap_withXtokensX
```

Comment:
Mentioned this earlier, but let's keep the `valueMap(true)` tests since we have to maintain that deprecated code. There are other places as well obviously where the `valueMap(true)` tests need to be brought back. Just duplicate the test basically - one for the old way and one for the new way.

Reply:
I kept the boolean overload tests in `ValueMapTest`. I don't think `AddVertexTest` should be used to test several `valueMap` overloads, but ... I'll add them back.

Reply:
ok - as long as we at least exercise the old stuff sufficiently then no need to bring them all back. I just saw these removed and didn't realize that there were others still hanging about.